website/content/docs/secrets/databases/redis.mdx

```diff
@@ -0,0 +1,124 @@
+---
+layout: docs
+page_title: Redis - Database - Secrets Engines
+description: |-
+  Redis is one of the supported plugins for the database secrets engine.
+  This plugin generates database credentials dynamically based on configured
+  roles for the Redis database, and also supports [Static Roles](https://developer.hashicorp.com/vault/docs/secrets/databases#static-roles).
+---
+
+# Redis Database Secrets Engine
+
+Redis is one of the supported plugins for the database secrets engine. This
+plugin generates database credentials dynamically based on configured roles for
+the Redis database.
+
+See the [database secrets engine](/docs/secrets/databases) docs for
+more information about setting up the database secrets engine.
+
+## Capabilities
+
+| Plugin Name                 | Root Credential Rotation | Dynamic Roles | Static Roles | Username Customization |
+| --------------------------- | ------------------------ | ------------- | ------------ | ---------------------- |
+| `redis-database-plugin`     | Yes                      | Yes           | Yes          | No                     |
+
+## Setup
+
+1.  Enable the database secrets engine if it is not already enabled:
+
+    ```shell-session
+    $ vault secrets enable database
+    Success! Enabled the database secrets engine at: database/
+    ```
+
+    By default, the secrets engine will enable at the name of the engine. To
+    enable the secrets engine at a different path, use the `-path` argument.
+
+1.  Configure Vault with the proper plugin and connection configuration:
```

Comment:
Should this be incrementing between sections?

Reply:
Hmm seems like this is like this in most places and only a handful like [MySQL](http://localhost:3000/docs/secrets/databases/mysql-maria) increment correctly. Could this be a general bug that was missed some time ago?

Seems odd to me that numerical bullet points all use 1.